migrations/1676279335613-AddCreateToolEtherpadPermissionForStudents.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose');
```

Comment:
not sure why we need 2 separate migrations?
These migration run without having option to say run only in one particular instance or not. So, no point in splitting them in my opinion....

Reply:
They are really performed in one specific case. 
For example, when a teacher previously had TOOL_CREATE permission, he could create an etherpad within a group, but a student could not. He lacked this permission. Then when I added this permission to the student, the student started to have too many rights. He could create and add things that he shouldn't be able to. 
Now with these two migrations, everything works like this:

1) The teacher creates an etherpad inside the course. The student cannot do this. He does not have the rights.

When I add TOOL_CREATE_ETHERPAD for the student, but take it away from the teacher and give the teacher TOOL_CREATE, the teacher cannot create an etherpad. 
Therefore, TOOL_CREATE_ETHERPAD is a special permission. 

2) The teacher can create video conferences, but the student cannot. 
3) The teacher creates an ethrpad within the group. The student can do this too. They have the rights to do so.

Reply:
ok, I'll split it